fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 org = "ballerinax"
 name = "health.fhirr4"
 version = "${project.version}"
-distribution = "2201.4.1"
+distribution = "2201.5.0"
```

Comment:
let's take this as well from the properties.

Reply:
Done